build.gradle.kts

```diff
@@ -0,0 +1,132 @@
+import com.github.jengelman.gradle.plugins.shadow.tasks.ShadowJar
+
+plugins {
+    `java-library`
+    checkstyle
+
+    id("net.minecrell.plugin-yml.bukkit") version "0.6.0"
+    id("com.github.johnrengelman.shadow") version "8.1.1"
+    id("xyz.jpenilla.run-paper") version "2.1.0"
+}
+
+group = "com.eternalcode"
+version = "1.0.0"
+description = "LobbyHeads"
+
+repositories {
+    gradlePluginPortal()
+    mavenCentral()
+
+    maven { url = uri("https://repo.panda-lang.org/releases") }
+    maven { url = uri("https://hub.spigotmc.org/nexus/content/repositories/snapshots/") }
+    maven { url = uri("https://jitpack.io") }
+    maven { url = uri("https://repo.extendedclip.com/content/repositories/placeholderapi/") }
+    maven { url = uri("https://storehouse.okaeri.eu/repository/maven-public/") }
+    maven { url = uri("https://repository.minecodes.pl/releases") }
+    maven { url = uri("https://libraries.minecraft.net/") }
+    maven { url = uri("https://repo.papermc.io/repository/maven-public/") }
+
+}
+
+checkstyle {
+    toolVersion = "10.12.3"
+
+    configFile = file("${rootDir}/checkstyle/checkstyle.xml")
+
+    maxErrors = 0
+    maxWarnings = 0
+}
+
+dependencies {
+    // okaeri configs
+    implementation("eu.okaeri:okaeri-configs-yaml-snakeyaml:5.0.0-beta.5")
+    implementation("eu.okaeri:okaeri-configs-serdes-commons:5.0.0-beta.5")
+
+    // a cool library, kyori
+    implementation("net.kyori:adventure-platform-bukkit:4.3.0")
+    implementation("net.kyori:adventure-text-minimessage:4.14.0")
+
+    // Rollczi's skullapi
+    implementation("dev.rollczi:liteskullapi:1.3.0")
+
+    // spigot-api
+    compileOnly("org.spigotmc:spigot-api:1.20.1-R0.1-SNAPSHOT")
+    testImplementation("org.spigotmc:spigot-api:1.20.1-R0.1-SNAPSHOT")
+
+    // mojang's authlib
+    compileOnly("com.mojang:authlib:1.6.25")
```

Comment:
The latest version of authlib found in my %appdata%/.minecraft/libraries is 5.0.47
![image](https://github.com/EternalCodeTeam/LobbyHeads/assets/77227023/b6ddd9c7-8328-4a16-8c40-e3caeb82aa4c)